devops/tasks.py

```diff
@@ -209,3 +224,198 @@ def release(
+
+    :param str env: The environment
+    """
+    settings = load_env_settings(env)
+    ensure_context(settings.KUBE_CONTEXT)
+
+    label(logger.info, f"Getting master key for {env}")
+
+    # Based on:
+    # https://github.com/bitnami-labs/sealed-secrets#how-can-i-do-a-backup-of-my-sealedsecrets
+    result = run(
+        [
+            "kubectl",
+            "get",
+            "secret",
+            "-n",
+            "kube-system",
+            "-l",
+            "sealedsecrets.bitnami.com/sealed-secrets-key",
+            "-o",
+            "yaml",
+        ]
+    )
+
+    content = result.stdout.decode(encoding="utf-8")
+    output_file = master_key_path(env=env)
+
+    logger.info(f"Saving master key to {output_file}")
+
+    with output_file.open("w", encoding="utf-8") as f:
+        f.write(content)
+
+
+def unseal_secrets(env: str) -> None:
+    """
+    Decrypts the secrets for the desired env and base64 decodes them to make
+    them easy to edit.
+
+    :param str env: The environment.
+    """
+    # Validate env
+    load_env_settings(env)
+
+    master_key = master_key_path(env=env)
+    if not master_key.exists():
+        get_master_key(env=env)
+
+    sealed_secret_files = [
+        secret_file
+        for secret_file in (Path("envs") / env / "secrets").glob("*.yaml")
+        if not secret_file.name.endswith(UNSEALED_SECRETS_EXTENSION)
+    ]
+
+    label(logger.info, f"Unsealing secrets for {env}")
+
+    for input_file in sealed_secret_files:
+        output_file = input_file.with_name(input_file.stem + UNSEALED_SECRETS_EXTENSION)
+
+        logger.info(f"Unsealing {input_file} to {output_file}")
+
+        with input_file.open("r", encoding="utf-8") as f:
+            content = f.read()
+
+        content = kube_unseal(content, master_key)
+        content = base64_decode_secrets(content)
+
+        with output_file.open("w", encoding="utf-8") as f:
+            f.write(content)
+
+        input_file.unlink()
+
+
+def seal_secrets(env: str) -> None:
+    """
+    Base64 encodes and seals the secrets for the desired env.
+
+    :param str env: The environment.
+    """
+    # Validate env
+    load_env_settings(env)
+
+    secrets_pem = secrets_pem_path(env=env)
+
+    unsealed_secret_files = (Path("envs") / env / "secrets").glob(
+        f"*{UNSEALED_SECRETS_EXTENSION}"
+    )
+
+    label(logger.info, f"Sealing secrets for {env}")
+
+    for input_file in unsealed_secret_files:
+        output_file_name = input_file.name[: -len(UNSEALED_SECRETS_EXTENSION)] + ".yaml"
+        output_file = input_file.with_name(output_file_name)
+
+        logger.info(f"Sealing {input_file} as {output_file}")
+
+        with input_file.open("r", encoding="utf-8") as f:
+            content = f.read()
+
+        content = base64_encode_secrets(content)
+        content = kube_seal(content, cert=secrets_pem)
+
+        with output_file.open("w", encoding="utf-8") as f:
+            f.write(content)
+
+        input_file.unlink()
```

Comment:
In this case, deleting the unsealed files makes more sense though :)

Reply:
To be consistent with above, also changed this to not delete. It's also possible that you don't have access to the master.key, in which case deleting the file could remove your only copy of the info, so better to leave the files.